capstone/src/main/java/com/google/sps/servlets/StudySetServlet.java

```diff
@@ -58,6 +60,11 @@
           search_sql_statement
         )
       ) {
+        String new_search_word = "%" + search_word + "%";
```

Comment:
you can probably inline this with L65

queryStatement.setString(i, "%" + search_word + "%");

Reply:
Done